### ISSUE TITLE
Error: This expression has type 'void' and can't be used.

### ISSUE THREAD

**Issue by kwalrath** [block-forest/dart-coveralls#81]
The stagehand travis build is failing due to dart_coveralls issues (https://travis-ci.org/dart-lang/stagehand/jobs/406415905):

```
Failed to precompile dart_coveralls:dart_coveralls:
file:///home/travis/.pub-cache/hosted/pub.dartlang.org/dart_coveralls-0.6.0+1/bin/src/report.dart:16:29: Error: This expression has type 'void' and can't be used.
    if (res["help"]) return print(parser.usage);
                            ^
file:///home/travis/.pub-cache/hosted/pub.dartlang.org/dart_coveralls-0.6.0+1/bin/src/report.dart:22:38: Error: This expression has type 'void' and can't be used.
    if (res.rest.length != 1) return print("Please specify a test file to run");
                                     ^
file:///home/travis/.pub-cache/hosted/pub.dartlang.org/dart_coveralls-0.6.0+1/bin/src/report.dart:25:36: Error: This expression has type 'void' and can't be used.
    if (!file.existsSync()) return print("Dart file does not exist");
                                   ^
file:///home/travis/.pub-cache/hosted/pub.dartlang.org/dart_coveralls-0.6.0+1/bin/src/upload.dart:15:29: Error: This expression has type 'void' and can't be used.
    if (res["help"]) return print(parser.usage);
                            ^
file:///home/travis/.pub-cache/hosted/pub.dartlang.org/dart_coveralls-0.6.0+1/bin/src/upload.dart:22:14: Error: This expression has type 'void' and can't be used.
      return print("Please specify a directory containing VM coverage files");
             ^
file:///home/travis/.pub-cache/hosted/pub.dartlang.org/dart_coveralls-0.6.0+1/bin/src/upload.dart:33:14: Error: This expression has type 'void' and can't be used.
      return print("Directory containing VM coverage files does not exist");
             ^
```

For now, I'll just skip anything in the stagehand build that requires dart_coveralls.

**Comment by pq** [comment]
This is hitting the linter too: https://github.com/dart-lang/linter/issues/1093.

I believe the current plan is to hold tight for a new SDK build w/ the breaking change reverted.

/cc @MichaelRFairhurst

**Comment by pq** [comment]
This is fixed.

Unfortunately, we still have failures due to stale `coverage` package deps (#83).

I've got a PR to fix: #82.

@nilsdoehring: could you take a look?

**Comment by pq** [comment]
Fixed in the latest dev SDK.
